backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglGraphics.java

```diff
@@ -79,7 +80,7 @@
 	}
 
 	LwjglGraphics (Canvas canvas, LwjglApplicationConfiguration config) {
-		this.config = config;
+		this(config);
```

Comment:
@intrigus It really doesn't matter either way, seeing as there are only two field assignments. If, however, there were more occurring in the constructors, it would make much more sense to re-use them. This is a common pattern, both in this library and in general (as long as you don't abuse it by telescoping params, in which case you should consider using a builder).